roles/patch-vulns/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Remove known log4j files / folders
+  become: yes
+  become_user: root
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ grid_home }}/tfa/{{ inventory_hostname_short }}/tfa_home/jlib/log4j-core-2.9.1.jar"
+    - "{{ grid_home }}/tfa/{{ inventory_hostname_short }}/tfa_home/jlib/tfa.war"
+    - "{{ grid_home }}/suptools/tfa/release/tfa_home/jlib/log4j-core-2.9.1.jar"
+    - "{{ grid_home }}/suptools/tfa/release/tfa_home/jlib/tfa.war"
+    - "{{ grid_home }}/.patch_storage"
+    - "{{ oracle_home }}/suptools/tfa/release/tfa_home/jlib/log4j-core-2.9.1.jar"
+    - "{{ oracle_home }}/suptools/tfa/release/tfa_home/jlib/tfa.war"
+    - "{{ oracle_home }}/md/property_graph/lib"
+    - "{{ oracle_home }}/md/property_graph/pgx"
+    - "{{ oracle_home }}/.patch_storage"
+    - "{{ swlib_path }}/V982063-01.zip"
+    - "{{ swlib_path }}/V982068-01.zip"
+  ignore_errors: true
```

Comment:
Do we need to ignore errors?  I think state: absent should succeed even if the file doesn't exist.

Reply:
Done, removed the line